index.css

```diff
@@ -0,0 +1,457 @@
+}
+
+.side-nav input[type="checkbox"] {
+  opacity: 0;
+  position: absolute;
+  z-index: -9999;
+}
+
+.side-nav svg {
+  width: 2.4rem;
+  height: 2.4rem;
+  transition: all .3s ease-in-out;
+}
+
+.side-nav label {
+  position: fixed;
+  top: 1rem;
+  left: 0;
+  padding: 1rem;
+  background: linear-gradient(to bottom right,#F8F3DF 50%,#DD7F77 50%);
+  color: var(--color-grey);
+  opacity: .9;
+  transition: all .3s ease-in-out;
+  z-index: 1
+}
+
+.side-nav input:focus + label {
+  outline: 3px solid var(--color-blue);
+}
+
+.side-nav input:checked + label {
+  left: 25rem;
+}
+
+.side-nav input:checked + label svg {
+  transform: rotate(90deg);
+}
+
+.side-nav input:checked ~ nav {
+  left: 0;
+}
+
+.side-nav input:not(:checked) ~ nav a:focus {
+  transform: translateX(25rem);
+}
+
+nav ul a {
+  padding: 1.6rem;
+  display: block;
+  text-decoration: none;
+  background: var(--color-white);
+  transition: .3s all ease-in-out;
+}
+
+nav ul a:hover,
+nav ul a:focus {
+  background: #dfdfdf;
+}
+
+nav li > ul a {
+  padding-left: 3rem;
+}
+
+nav li > ul a:hover,
+nav li > ul a:focus {
+  background: #efefef;
+}
+
+/*-- Media Items --*/
+.media-items {
+  padding: 2.6rem 1.6rem;
+  display: -webkit-flex;
+  display: -ms-flex;
+  display: flex;
+  -webkit-flex-direction: row;
+  -ms-flex-direction: row;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  box-sizing: border-box;
+  background: linear-gradient(to bottom right,#516B5E,#367588);
+}
+
+.media-items h1 {
+  width: 100%;
+  text-align: center;
+  color: var(--color-white);
+}
+
+.media-items article {
+  position: relative;
+  max-width: 40rem;
+  max-height: 25rem;
+  display: -webkit-flex;
+  display: -ms-flex;
+  display: flex;
+  -webkit-flex-direction: column;
+  -ms-flex-direction: column;
+  flex-direction: column;
+  justify-content: center;
+  -ms-align-items: center;
+  align-items: center;
+  overflow: hidden;
+  margin: 1.6rem;
+  transition: all .3s ease-in-out;
+}
+
+.media-items img {
+  width: 100%;
+}
+
+.media-items a {
+  position: absolute;
+  text-align: center;
+  padding: 1.6rem;
+  box-sizing: border-box;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  display: -webkit-flex;
+  display: -ms-flex;
+  display: flex;
+  -webkit-flex-direction: column;
+  -ms-flex-direction: column;
+  flex-direction: column;
+  justify-content: center;
+  background: linear-gradient(to bottom right,hsla(0, 0%, 100%, .5),hsla(214, 50%, 50%, .5));
+  backdrop-filter: blur(0);
+  -webkit-backdrop-filter: blur(0);
+  opacity: 0;
+  transition: all .3s ease-in-out;
+  text-decoration: none;
+}
+
+.media-items article:hover,
+.media-items article:active {
+  box-shadow: var(--main-shadow);
+  transform: translateY(-5px);
+}
+
+.media-items article:hover > a,
+.media-items article:active > a,
+.media-items a:focus {
+  backdrop-filter: blur(5px);
+  -webkit-backdrop-filter: blur(5px);
+  opacity: 1;
+}
+
+/*-- Login -- */
+#login {
```

Comment:
Try to defer using id selectors, specificity can and will f*ck you over in the long run